api-migration.md

```diff
@@ -15,6 +15,27 @@ The library continues to support legacy plugins. They can be used in conjunction
 with new plugins. Your plugins list may contain any number of new plugins but
 only ever one legacy plugin.
 
+It's important to use the response object returned by the event hook when
```

Comment:
What about merging the two sentences to make it clear that only the response object is available when piping and not the request object.  Something like: `It's important to note that cloudant.request object is a pass through stream and only the response object returned by the event hook is usable when piping`

Reply:
Thanks, I've had another bash at this. Hopefully it makes a little more sense.